vwifi.c

```diff
@@ -554,52 +696,64 @@ static struct owl_context *owl_create_context(void)
      */
     ret->wiphy->signal_type = CFG80211_SIGNAL_TYPE_MBM;
 
+    ret->wiphy->flags |= WIPHY_FLAG_NETNS_OK;
+
     /* register wiphy, if everything ok - there should be another wireless
      * device in system. use command: $ iw list
      * Wiphy owl
      */
     if (wiphy_register(ret->wiphy) < 0)
         goto l_error_wiphy_register;
 
-    /* allocate network device context. */
-    ret->ndev =
-        alloc_netdev(sizeof(*ndev_data), NDEV_NAME, NET_NAME_ENUM, ether_setup);
-    if (!ret->ndev)
-        goto l_error_alloc_ndev;
+    INIT_LIST_HEAD(&ret->netintf_list);
+    for (int i = 0; i < NDEV_NUMS; i++) {
+        /* allocate network device context. */
+        struct net_device *ndev = NULL;
+        ndev = alloc_netdev(sizeof(struct owl_ndev_priv_context),
+                            i == 0 ? NDEV_NAME : SINKDEV_NAME, NET_NAME_ENUM,
+                            ether_setup);
+        if (!ndev)
+            goto l_error_alloc_ndev;
+        /* fill private data of network context. */
+        ndev_data = ndev_get_owl_context(ndev);
+        ndev_data->owl = ret;
+        ndev_data->ndev = ndev;
+        /* fill wireless_dev context.
```

Comment:
Add proper blank line(s) to emphasize the comments and the correspond code snips.